magefiles/magefile.go

```diff
@@ -354,7 +354,7 @@ func (ci CI) setRequiredEnvVars() error {
 			case strings.Contains(jobName, "image-controller"):
 				envVarPrefix = "IMAGE_CONTROLLER"
 				imageTagSuffix = "image-controller-image"
-				testSuiteLabel = "image-controller"
+				testSuiteLabel = "build"
```

Comment:
running the whole build suite probably doesn't make sense + none of the "build" tests uses image-controller.

I think we could add the image-controller annotation [here](https://github.com/redhat-appstudio/e2e-tests/blob/c309ba85e6bccc99e7ba6445ee3b3fc584dd10b9/pkg/utils/has/controller.go#L243), which is the method for creating a component used by ["pac-build" test](https://github.com/redhat-appstudio/e2e-tests/blob/c309ba85e6bccc99e7ba6445ee3b3fc584dd10b9/tests/build/build.go#L36), wdyt?

Reply:
@psturc Added the image-controller annotations during component creation for PaC enabled builds. Also used "pac-build" label when running the tests for image-controller repo.